src/shared/cloudformation/activation.ts

```diff
@@ -56,12 +56,12 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
  * and slowing down the extension starting up.
  */
 function setTemplateRegistryInGlobals(registry: CloudFormationTemplateRegistry) {
-    const registrySetupFunc = async (registry: CloudFormationTemplateRegistry) => {
-        await registry.addExcludedPattern(CloudFormation.devfileExcludePattern)
-        await registry.addExcludedPattern(CloudFormation.templateFileExcludePattern)
-        await registry.addWatchPatterns([CloudFormation.templateFileGlobPattern])
-        await registry.watchUntitledFiles()
-
+    const registrySetupFunc = async (registry: CloudFormationTemplateRegistry, cancellationPromise: Promise<void>) => {
+        registry.addExcludedPattern(CloudFormation.devfileExcludePattern)
+        registry.addExcludedPattern(CloudFormation.templateFileExcludePattern)
```

Comment:
why remove `await`? `addExcludedPattern` calls rebuild() which calls `findFiles`, surely we don't benefit if those are run in a random order.

It might not matter for the current implementation, but if async functions are unawaited that signals that they don't have shared state and are safe to be scheduled however the node event loop or OS chooses to. But if there's no advantage to doing so, then the lack of `await` is a cognitive burden, because it means we need to be sure that this assumption continues to be true.